cura/Arranging/ArrangeObjectsJob.py

```diff
@@ -28,18 +28,20 @@ def run(self):
                                  title = i18n_catalog.i18nc("@info:title", "Finding Location"))
         status_message.show()
 
-        found_solution_for_all = None
```

Comment:
Maybe initialize the variable with False and don't set it a the exception block
```suggestion
        found_solution_for_all = False
```